src/Components/Steps/ZipcodeStep.tsx

```diff
@@ -164,7 +167,7 @@ export const ZipcodeStep = () => {
                           id="questions.zipcode-a-disabledSelectMenuItemText"
                           defaultMessage="Select a county"
                         />,
-                        countiesByZipcode[watch('zipcode')],
+                        countiesByZipcode[parsedZipCode.data],
```

Comment:
What do you think about using the original expression? I understand that there are performance trade offs with using `watch` in line, but it's more readable and explicit in terms of what the `zipcode` value is.

Reply:
The new way, removes the spaces, so that we don't get a key error.